android/src/test/java/io/github/droidkaigi/feeder/FeedItemViewModelTest.kt

```diff
@@ -116,8 +117,19 @@ class FeedItemViewModelTest(
                                     null
                                 }
                             ),
+                            feedItemDao = fakeFeedItemDao(
+                                if (errorFetchData) {
+                                    AppError.UnknownException(Throwable("Database Exception"))
+                                } else {
+                                    null
+                                }
+                            ),
                             dataStore = fakeUserDataStore()
-                        )
+                        ).also {
+                            runBlockingTest {
+                                if (!errorFetchData) it.refresh()
+                            }
+                        }
```

Comment:
Thanks for fixing the test too!
Why is this code needed? I tried erasing it and moving it, but the test seems to pass 👀

Reply:
Sorry, you are right 😅 
I forgot refreshing on ViewModel init.
I removed this 🙏